src/test/java/address/unittests/keybindings/KeyBindingsManagerApiTest.java

```diff
@@ -75,8 +75,6 @@ public void keyEventDetected_definedBinding_matchingEventRaised() throws Excepti
 
     @Test
     public void keyEventDetected_undefinedBinding_ignored(){
-        verifyIgnored("A");
-        verifyIgnored("G", "A");
```

Comment:
I'm going to merge this first so i can continue to do the next stage of multiple select. Let me know if there is any issue, and i will fix it in the next PR.